### DISCUSSION TITLE
"Ender3V2-422-BLTUBL-LA-MPC-20230904.bin" do not installing after "Ender3V2-422-BLTUBL-MPC-20230904.bin"

### DISCUSSION THREAD

**Discussion by ArtemShevtsov** [mriscoc/Ender3V2S1#1107] [Q&A]
After successful installation of Ender3V2-422-BLTUBL-MPC-20230904.bin on my "ender 3 v2 neo (GD32F303 RET6 chip b.t.w.)" I've figured out that to enable "linear advance" feature I need to use Ender3V2-422-BLTUBL-LA-MPC-20230904.bin.
So I did formatting of my flash card, copied "LA" bin file and give it to printer, but it is not installed :(
printer just loaded and saying that it has "Ender3V2-422-BLTUBL-MPC-20230904" software, without LA.

What am I doing wrong and how to make "LA" firmware works?

**Comment by mriscoc** [comment]
Remember to change the name of the firmware file in each time you try. Reformat the SD memory card. Try other cards. Pronterface connect to the Professional Firmware at 250000 baud

**Reply by ArtemShevtsov** [reply]
Tried to rename the file, did a few  triea with different names - same result :(
Also I did formatting before each try as FAT32 4096, did not help :(
It do not work trying to upload .bin for TJC display as well ( by inserting card into slot in display itself ).

Maybe different card will help, but I dont have it unfortunately.

PS. According to pronterface, thanks, I found that and edit my initial post.

**Reply by ArtemShevtsov** [reply]
What helped for me: is to use "Disk Management" tool in windows. "Delete Volume" first, and format with 8192 allocation size.
Now I have LA on my printer, thanks :)
But still, somehow  Ender3V2-422-BLTUBL-TJC-MPC-20230904.bin  from "tjc" tag does not applied for my display.
